src/components/Filters/FilterChecklist.vue

```diff
@@ -48,9 +48,11 @@
           <LicenseIcons v-if="filterType === 'licenses'" :license="item.code" />
           {{ itemLabel(item) }}
         </label>
+        
         <button class="appearance-none">
```

Comment:
There is currently a bug with this button: 
When you click, it reloads the page. This happens because a button inside the form is by default of type "submit", so it currently submits the form (and `stop` modifier on the `@click.stop` handler does not change it). 
Adding "type='button'" will fix this.